blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_9.py

```diff
@@ -0,0 +1,102 @@
+"""Formula 5.9 from NEN-EN 1993-5:2008 Chapter 5 - Ultimate limit state."""
+
+import numpy as np
+
+from blueprints.codes.eurocode.nen_en_1993_5_2008.chapter_5_ultimate_limit_states import NEN_EN_1993_5_2008
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula, fraction, min_curly_brackets
+from blueprints.type_alias import CM3, DEG, DIMENSIONLESS, KNM, MM, MM2, MPA
+from blueprints.unit_conversion import CM3_TO_MM3, NMM_TO_KNM
+from blueprints.validations import raise_if_less_or_equal_to_zero
+
+
+class Form5Dot9ReducedBendingMomentResistance(Formula):
+    """Class representing formula 5.9 for reduced design bending moment resistance of the cross-section."""
+
+    label = "5.9"
+    source_document = NEN_EN_1993_5_2008
+
+    def __init__(  # noqa: PLR0913
+        self,
+        beta_b: DIMENSIONLESS,  # Reduction factor for bending resistance
+        w_pl: CM3,  # Plastic section modulus
+        rho: DIMENSIONLESS,  # Reduction factor for shear resistance
+        a_v: MM2,  # Projected shear area for each web
+        t_w: MM,  # Thickness of the web
+        alpha: DEG,  # Inclination of the web
+        f_y: MPA,  # Yield strength
+        gamma_m_0: DIMENSIONLESS,  # Partial factor for material properties
+        mc_rd: KNM,  # Design moment resistance
+    ) -> None:
+        """(Mv,Rd) Calculate reduced design bending moment resistance of the cross-section allowing for the shear force in [kNm/m].
+
+         based on NEN-EN 1993-5:2007(E) art. 5.2.2(9) formula 5.9.
+
+        Parameters
+        ----------
+        beta_b : DIMENSIONLESS
+            (β_b) Reduction factor for the bending resistance of the cross-section in [-].
+            Defined in NEN-EN 1993-5:2007(E) art. 5.2.2(2) or CUR166, part 2, par. 3.3.2.
+        w_pl : CM3
+            (Wpl) Plastic section modulus in [cm³/m].
```

Comment:
why shouldn't we just request MM3 input for w_pl? all arguments are in mm, N/mm etc. and then we are changing that here.. if we just be explicit that we expect mm3 we don't need to do any conversion. I think that is much more straightforward.

I would also remove the '/m' ....

As far as I can find EC3-5 is not really explicit about which unit to use here. apart from this suggestion:

![image](https://github.com/Blueprints-org/blueprints/assets/102153534/93042dc6-236a-4e5a-b37f-814610e47db6)

Reply:
I agree
![afbeelding](https://github.com/Blueprints-org/blueprints/assets/44614728/629d2373-6de6-4a75-a90c-d6540cc84a11)